tests/console/rmt/rmt_proxy.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: yast2-rmt rmt-server
+# Summary: Add rmt configuration test and basic configuration via
+#    rmt-wizard, test RMT basic function behind proxy.
+# Maintainer: Lemon Li <leli@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use repo_tools;
+use utils;
+
+sub run {
+    select_console 'root-console';
+    record_info('RMT server setup', 'Start to setup a rmt server');
+    rmt_wizard();
+    # sync from SCC
+    rmt_sync;
+
+    # Set RMT behind a proxy (with NTLM Auth), and ensure RMT basic function works.
+    record_info('Set proxy', 'Set RMT behind a proxy (with NTLM Auth)');
+    # set the proxy setting to rmt.conf
+    my $proxy_url = 'http:\/\/' . get_required_var('PROXY_SERVER');
```

Comment:
What kind of configuration needed for this PROXY_SERVER?  
I suppose this case will be a multi machine setup?

Reply:
I don't think we must run multi-machine for this test, it just need a proxy server to support proxy service is enough.

Reply:
If other team use this module then we will face issue since no one know what PROXY_SERVER is and how configure it etc..

Reply:
Updated code to support multi machine test.